src/app/App.js

```diff
@@ -27,6 +29,7 @@ export default class App extends Component {
             <PrivateRoute path="/" exact component={SearchPage} />
             <PrivateRoute path="/search" component={ResultsPage} />
             <PrivateRoute path="/customers/:id/view" component={DetailsPage} />
+            <Route path="/tenancies/:id" component={TenancyDetailsPage} />
```

Comment:
Should the path here be `/tenancies/:id/view` for consistency with `/customers`? Personally I'm not a fan of the additional `/view`, but it was there first...

Reply:
I agree - I'm racking my brain why we had /view originally but can't think of a good reason. I think we should improve things and remove the trailing /view from the customers endpoint.